src/lib/atoms/buttons/UploadButton.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Papa from 'papaparse'  // http://papaparse.com/docs
+import ReactFileReader from 'react-file-reader'
+
+import { Button } from 'LibIndex'
+
+
+class UploadButton extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {}
+  }
+
+  handleFiles = (files) => {
+    const { config, handleResults } = this.props
+    const reader = new FileReader()
+
+    // TODO: How do we test this?
+    // NOTE: Currently, very hard to test.  Be aware
+    reader.onload = (e) => {
```

Comment:
This is currently impossible to test (At least I think so).  We should look into a better way to do this.